app/controllers/reviews_controller.rb

```diff
@@ -22,7 +19,7 @@ def new
   def review_params
     params.require(:review).permit(:title, :description, :rating).merge(
       show: Show.find(params[:show_id])).merge(
-      user: User.find(current_user)
+        user: User.find(current_user.id)
```

Comment:
Again -- `current_user` finds the instance of `User` and makes it available in memory. Why not write this as `user: current_user`?